README.md

```diff
@@ -47,8 +47,11 @@ You can also run the Client and Server on the same machine. When launching the c
 
 * To get started contributing to the project, see the [contributing guide](CONTRIBUTING.md).
 * Please also take a look at our [code of conduct](code-of-conduct.md), that acts as a guideline on how to communicate with people on this project. If you experienced a violation, do not hesitate to contact one of the maintainers.
-* We follow [these](https://google.github.io/styleguide/javaguide.html) Java coding conventions, so please make sure that you adhere to them.
-* Please format your code. This project considers the output of the [IntelliJ IDEA](https://www.jetbrains.com/idea/) IDE `Reformat Code` option as the gold standard.
+* We follow the [Google Java coding conventions](https://google.github.io/styleguide/javaguide.html), so please make sure that you adhere to them.
+* Please format your code.
+    * How to format code using [IntelliJ IDEA or Eclipse](https://github.com/google/google-java-format)
+    * How to format code using VSCode please see the file: VSCode_format_guide.md
```

Comment:
The markdown file for the `VSCode formatting guide` does not seem to be linked, like e.g. the `contributing guide` on line 47. Can you please change that?